Aliucord/src/main/java/com/aliucord/coreplugins/rn/Patches.kt

```diff
@@ -150,6 +159,24 @@ fun patchDefaultAvatars() {
     )
 }
 
+fun patchUsername() {
+    if (StoreStream.getUsers().me.discriminator == 0) {
```

Comment:
you could use an early return here instead to reduce nesting

Reply:
Oh yea I'll do that